packages/web-components/src/components/va-search-input/va-search-input.scss

```diff
@@ -0,0 +1,145 @@
+@forward 'settings';
+
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@use 'usa-search/src/styles/usa-search';
+@use 'usa-button/src/styles/usa-button';
+
+@import '../../mixins/uswds-error-border.scss';
+@import '../../global/formation_overrides';
+
+:host {
+  display: block;
+  border: none;
+  padding: 0;
+}
+
+:host([uswds]:not([uswds='false'])) .usa-input {
+  margin-top: 0;
+}
+
+:host([uswds]:not([uswds='false'])) #va-search-listbox {
+  top: 37px;
+  z-index: 1;
+}
+
+:host([uswds]:not([uswds='false'])[big="true"]) #va-search-listbox  {
+  top: 52px;
+}
+
+.usa-button {
+  line-height: 0;
+}
+
+.usa-input {
+  max-width: 100%;
+}
+
+/* if small prop is active, or, if mobile size is active, display va-icon search button */
+:host([small]:not([small='false'])) va-icon {
+  display: block;
+}
```

Comment:
A side-effect of using `va-icon` meant that I added the CSS display logic so that the search icon would always display on small viewports as well as when the small prop is set.